.github/dependabot.yml

```diff
@@ -67,16 +62,7 @@ updates:
   - package-ecosystem: "pip"
     # Cirq has requirements.txt files in multiple places. N.b. the use of
     # attribute "directories" instead of "directory" here.
-    directories:
-      - "/"
-      - "/cirq-aqt"
-      - "/cirq-core"
-      - "/cirq-google"
-      - "/cirq-ionq"
-      - "/cirq-pasqal"
-      - "/cirq-rigetti"
-      - "/cirq-web"
-      - "/dev_tools/requirements"
+    directory: "/dev_tools/requirements/deps"
```

Comment:
Please add to `cirq-all-no-contrib.txt` the following files:

- cirq-ionq/requirements.txt
- cirq-pasqal/requirements.txt
- cirq-web/requirements.txt

These are currently not included at all.

Reply:
> Please add to `cirq-all-no-contrib.txt` the following files:
> 
> * cirq-ionq/requirements.txt
> * cirq-pasqal/requirements.txt
> * cirq-web/requirements.txt
> 
> These are currently not included at all.

Will do. Just to clarify: do you mean to do it in a different PR, or as part of this? (Seems better to separate it, as it wouldn't be a dependabot issue per se.)

Reply:
> Just to clarify: do you mean to do it in a different PR, or as part of this?

In this PR please.  The change makes dependabot check those files, they would be left out otherwise.

As a side benefit, it will also future proof dev.env.txt by making it use those  files.  Currently it makes no difference, because dependencies they bring in are  present elsewhere, but that may change in the future.

Reply:
OK, done.